test/cluster/object_store/test_backup.py

```
@@ -771,43 +789,61 @@ async def test_restore_tablets(build_mode: str, manager: ScyllaClusterManager, o
+                manifests = [f'{s.server_id}/{cf}/manifest.json' for s in servers]
+                tid = await manager.api.restore_tablets(node.ip_addr, dst_ks, f'{cf}_restored', snap_name, servers[0].datacenter, object_storage.address, object_storage.bucket_name, manifests, restore_prefix)
+                status = await manager.api.wait_task(node.ip_addr, tid)
+                assert (status is not None) and (status['state'] == 'done'), f"Restore of {cf} via {node.ip_addr} failed: {status}"
+                assert status['progress_total'] > 0
+                assert status['progress_completed'] == status['progress_total']
+
+            await asyncio.gather(*(restore_table(i, cf) for i, cf in enumerate(tables)))
+
+            await asyncio.gather(*(check_mutation_replicas(cql, manager, servers, range(num_keys), topology, logger, dst_ks, f'{cf}_restored') for cf in tables))
+
+            res = await cql.run_async(f"SELECT extra FROM {dst_ks}.{tables[0]}_restored;")
```

Ruff (0.16.2)

[error] Possible SQL injection vector through string-based query construction

(S608)